docx/from/docxtotei.xsl

```diff
@@ -1,713 +1,743 @@
+  <xsl:variable name="docProps" select="concat($wordDirectory,'/docProps/core.xml')"/>
+  <xsl:variable name="numberFile" select="concat($wordDirectory,'/word/numbering.xml')"/>
+  <xsl:variable name="relsDoc" select="concat($wordDirectory,'/word/_rels/document.xml.rels')"/>
+  <xsl:variable name="relsFile"  select="concat($wordDirectory,'/word/_rels/document.xml.rels')"/>
+  <xsl:variable name="styleDoc" select="concat($wordDirectory,'/word/styles.xml')"/>
+  <xsl:strip-space elements="*"/>
+  <xsl:preserve-space elements="w:t"/>
+  <xsl:output method="xml" version="1.0" encoding="UTF-8" indent="no"/>
+
+  <doc xmlns="http://www.oxygenxml.com/ns/doc/xsl">
+    <desc>
+      <p>The main template that starts the conversion from docx to TEI</p>
+      <p><b>IMPORTING STYLESHEETS AND OVERRIDING MATCHED TEMPLATES:</b></p>
+
+      <p>When importing a stylesheet (xsl:import) all the templates
+        in the imported stylesheet get a lower import-precedence than
+        the ones in the importing stylesheet. If the importing
+        stylesheet wants to override, let's say a general template to
+        match all &lt;w:p&gt; elements where no more specialized rule
+        applies it can't, since it will automatically override all
+        w:p[somepredicate] template in the imported stylesheet as
+        well.  In this case we have outsourced the processing of the
+        general template into a named template and all the imported
+        stylesheet does is to call the named template. Now, the
+        importing stylesheet can simply override the named template,
+        and everything works out fine.</p>
+
+      <p>See templates: - w:p (mode: paragraph)</p>
+
+      <p>Modes:</p>
+      <ul>
+        <li>pass0: a normalization process for styles. Can also
+          detect illegal styles.</li>
+
+        <li>pass2: templates that apply in the second stage
+          of the conversion, cleaning TEI elements created in the
+          first ise."</li>
+
+        <li>inSectionGroup: defines a template that works on a
+          group of consecutive elements (w:p or w:tbl elements) that
+          form a section (a normal section, not to be confused with
+          w:sectPr).</li>
+
+        <li>paragraph: defines that the template
+          works on an individual element (usually
+          starting with a w:p element).  </li>
+
+        <li>iden: simply copies the content</li>
+      </ul>
+
+    </desc>
+  </doc>
+  <xsl:template match="/">
+    <!-- Do an initial normalization and store everything in $pass0 -->
+    <xsl:if test="not(doc-available($relsFile))">
+      <xsl:message terminate="yes">The file <xsl:value-of
+          select="$relsFile"/> cannot be read</xsl:message>
+    </xsl:if>
+    <xsl:if test="not(doc-available($styleDoc))">
+      <xsl:message terminate="yes">The file <xsl:value-of
+          select="$styleDoc"/> cannot be read</xsl:message>
+    </xsl:if>
+
+    <xsl:variable name="passOneHalf">
```

Comment:
Yes, we could rename them.  But I like keeping the consistency in the names between these stylesheets and the source. It'll make it easier to port fixes over from the base.

Though I suppose I blew that away when I formatted the document.  Humm.